lib/generators/react_on_rails/setup_files_generator.rb

```diff
@@ -0,0 +1,38 @@
+require "rails/generators"
+
+module ReactOnRails
+  module Generators
+    class SetupFilesGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def create_assets_symlinks
+        create_link "client/assets/fonts", "../../app/assets/fonts" if Dir.exist?("app/assets/fonts")
+        create_link "client/assets/images", "../../app/assets/images" if
+        Dir.exist?("app/assets/images")
+      end
+
+      def create_jsx_files
+        copy_file "client/app/startup/clientGlobals.jsx",
+                  "client/app/startup/clientGlobals.jsx"
+        copy_file "client/app/startup/serverGlobals.jsx",
+                  "client/app/startup/serverGlobals.jsx"
+      end
+
+      def update_git_ignore
+        append_to_file ".gitignore" do
+          "# React on Rails\nnpm-debug.log\nnode_modules\n\n# Generated js bundles\n/app/assets/javascripts/generated/*"
+        end
+      end
+
+      def update_application_js
+        append_to_file "app/assets/javascripts/application.js" do
```

Comment:
Awesome, thanks for the feedback. I'll will test this out now and update the method.